Gulpfile.js

```diff
@@ -0,0 +1,36 @@
+var gulp = require('gulp'),
```

Comment:
We should use double quotes throughout this file, [per our style guide](https://github.com/thoughtbot/guides/tree/master/style/javascript).